src/org/opensolaris/opengrok/authorization/AuthorizationFramework.java

```diff
@@ -275,12 +297,14 @@ private String getClassName(JarEntry f) {
      * Old instances of plugins are removed and new list of plugins is
      * constructed. Unload and load event is fired on each plugin.
      *
-     * @see IAuthorizationPlugin#load() 
-     * @see IAuthorizationPlugin#unload() 
+     * @see IAuthorizationPlugin#load()
+     * @see IAuthorizationPlugin#unload()
      */
     @SuppressWarnings("unchecked")
     public synchronized void reload() {
         if (directory == null || !directory.isDirectory() || !directory.canRead()) {
+            LOGGER.log(Level.INFO, "plugin directory not found or not readable: {0}. "
```

Comment:
ERROR level would be better I think - if someone configures the directory in config, he probably wants to use it so treating as error is in order.